src/index.ts

```diff
@@ -45,6 +45,13 @@ import { WorkerEntrypoint } from 'cloudflare:workers';
 
 import { BaseRpcOptions, IssueOptions } from './types';
 
+export {
```

Comment:
any reason to not?
```suggestion
export * from './errors'
```

Reply:
There's a lot of stuff there that doesn't matter to users of this as an RPC target. Like "method not allowed" as well as the "handleError" function